deployment/postgres/configmap.yaml

```
@@ -0,0 +1,10 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: postgres-configmap
+data:
+  DATABASE_NAME: postgres
+  DATABASE_URL: jdbc:postgresql://localhost:5433/postgres
+  DATABASE_SCHEMA: public
+  SPRING_DATASOURCE_USERNAME: "admin"
+  SPRING_DATASOURCE_PASSWORD: "password"
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)